File: CS/OrgChartBindingExample/ViewModels/MainViewModel.cs
using DevExpress.Mvvm;
using OrgChartBindingExample.Model;
using System.Collections.Generic;

namespace OrgChartBindingExample.ViewModels {
    public class MainViewModel : ViewModelBase {

        public IList<Contact> Contacts { get; protected set; }

        public MainViewModel() {

            if (!IsInDesignMode)
                InitializeInRuntime();
            else
                InitializeInDesingMode();
        }
        void InitializeInRuntime() {

            Contacts = new ContactContextInitializer().Generate();
        }
        void InitializeInDesingMode() {
            Contacts = new List<Contact>() {
                new Contact("FirstName", "LastName") {
                    Gender = Gender.Female, Email = "email", Phone = "(555)555-0000",
                    Address = "Address", City = "City", State = "AA", Zip = "11111",
                },
            };
        }
    }
}

File: VB/OrgChartBindingExample/ViewModels/MainViewModel.vb
Imports DevExpress.Mvvm
Imports OrgChartBindingExample.Model
Imports System.Collections.Generic

Namespace OrgChartBindingExample.ViewModels

    Public Class MainViewModel
        Inherits ViewModelBase

        Private _Contacts As IList(Of OrgChartBindingExample.Model.Contact)

        Public Property Contacts As IList(Of Contact)
            Get
                Return _Contacts
            End Get

            Protected Set(ByVal value As IList(Of Contact))
                _Contacts = value
            End Set
        End Property

        Public Sub New()
            If Not IsInDesignMode Then
                InitializeInRuntime()
            Else
                InitializeInDesingMode()
            End If
        End Sub

        Private Sub InitializeInRuntime()
            Contacts = New ContactContextInitializer().Generate()
        End Sub

        Private Sub InitializeInDesingMode()
            Contacts = New List(Of Contact)() From {New Contact("FirstName", "LastName") With {.Gender = Gender.Female, .Email = "email", .Phone = "(555)555-0000", .Address = "Address", .City = "City", .State = "AA", .Zip = "11111"}}
        End Sub
    End Class
End Namespace
